source/_components/climate.tesla.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: "Tesla HVAC"
+description: "Instructions how to integrate Tesla climate system (HVAC) into Home Assistant."
+date: 2017-08-30 12:20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
+ha_category: Climate
+ha_iot_class: "Cloud push"
+ha_release: 0.53
+---
+
+The `Tesla` climate platform allows you to control your [Tesla](https://www.tesla.com/) climate from within Home Assistant.
+
+They will be automatically discovered if the Tesla component is loaded.
```

Comment:
The climate platform will be automatically configured if [Tesla](link) component is configured.

Reply:
`if the Tesla component`